chapters/equations.tex

```diff
@@ -817,12 +817,18 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
 \lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
-automatically such that the resulting system is structurally nonsingular.  In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user
-expects.  A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without
-informing the user.  For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly
-used on these variables, do not have an effect on the simulation.
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.
 \end{nonnormative}
+Based on these non-normative considerations we have the following rules:
+A tool may add or remove initial equations automatically such that the resulting system is structurally nonsingular.
+In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user expects.
+\begin{itemize}
+\item A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without informing the user.
+For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly used on these variables, do not have an effect on the simulation.
+\item A \lstinline!start!-attribute that is not fixed may be treated as fixed with a diagnostics.
+\item A consistent \lstinline!start!-value or initial equation may be removed with a message/warning.
```

Comment:
This `start`-value is still hyphenated.

Reply:
Comment on the suggested change by @henrikt-ma: "diagnostics" is plural so should not use an article "a". 
```suggestion
\item A consistent \lstinline!start!-value or initial equation may be removed with diagnostics.
```
Ugh, a quick search for "diagnostic" in the specification shows a couple of other instances where we say "a diagnostics is".

Reply:
Yeah, it wasn't typo by accident; rather a deliberate typo. :)  I've always found this use of _diagnostics_ strange.

Reply:
Looking around in the document, it looks as if we can resolve this in most or all places by just removing the _a_ in front of _diagnostics_.

Reply:
It would make more sense to me to use the singular when we mean a warning should be given. I have always been a bit uncertain about what is expected from the tool when it says _diagnostics must be given_. It feels like I need to do something more than just issue a warning and I am not sure what it is supposed to be.

Reply:
Yes, maybe that's the least confusing wording.  For the, the intention of saying _diagnostics_ without the _a_ in front is to convey the same kind of uncountable sense as in _information_ – we don't want to get into details about how much diagnostic information should be given to the user, as long as there is some.  But hey, if that is not the right way of saying it, my good intentions are nothing worth.

Another reason to say _a diagnostic_ is that it becomes consistent with the stronger _a warning_ – to just say _warnings_ when meaning typically a single warning sounds wrong even to me. :)